cloudbuild/cloudbuild.yaml

```diff
@@ -53,60 +53,60 @@ steps:
       - 'CLOUDSDK_COMPUTE_REGION=us-central1'
       - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
 
-  # Run presubmit tests in parallel for 2.0 Rocky Linux image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-2.0-rocky8-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '2.0-rocky8']
-    env:
-      - 'COMMIT_SHA=$COMMIT_SHA'
-      - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-      - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
-  # Run presubmit tests in parallel for 2.0 Ubuntu image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-2.0-ubuntu18-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '2.0-ubuntu18']
-    env:
-      - 'COMMIT_SHA=$COMMIT_SHA'
-      - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-      - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
-# Run presubmit tests in parallel for 2.1 Debian image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-2.1-debian11-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '2.1-debian11']
-    env:
-    - 'COMMIT_SHA=$COMMIT_SHA'
-    - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-    - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
-# Run presubmit tests in parallel for 2.1 Rocky Linux image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-2.1-rocky8-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '2.1-rocky8']
-    env:
-    - 'COMMIT_SHA=$COMMIT_SHA'
-    - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-    - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
-
-# Run presubmit tests in parallel for 2.1 Ubuntu image
-  - name: 'gcr.io/cloud-builders/kubectl'
-    id: 'dataproc-2.1-ubuntu20-tests'
-    waitFor: ['gcr-push']
-    entrypoint: 'bash'
-    args: ['cloudbuild/run-presubmit-on-k8s.sh', 'gcr.io/$PROJECT_ID/init-actions-image:$BUILD_ID', '$BUILD_ID', '2.1-ubuntu20']
-    env:
-    - 'COMMIT_SHA=$COMMIT_SHA'
-    - 'CLOUDSDK_COMPUTE_REGION=us-central1'
-    - 'CLOUDSDK_CONTAINER_CLUSTER=init-actions-presubmit'
+#  # Run presubmit tests in parallel for 2.0 Rocky Linux image
```

Comment:
again, leave the comments for posterity for one release and then remove them to reduce file size and execution time